aiotruenas_client/websockets/machine.py

```diff
@@ -35,16 +36,48 @@ def __init__(self) -> None:
 
     @classmethod
     async def create(
-        cls, host: str, password: str, username: str = "root", secure: bool = True
+        cls,
+        host: str,
+        password: Optional[str] = None,
+        username: Optional[str] = "root",
```

Comment:
It's probably best to have this default to `None` now, since we have two different authentication ways in this method now.

Reply:
I made this None and reordered the parameters since api_key is the preferred.